rails_multitenant.gemspec

```diff
@@ -17,17 +17,16 @@ Gem::Specification.new do |spec|
   spec.test_files    = Dir.glob('spec/**/*')
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = '>= 2.1.0'
+  spec.required_ruby_version = '>= 2.4.0'
 
-  spec.add_development_dependency "bundler", "~> 1.10"
-
-  spec.add_dependency 'activerecord', ENV.fetch('RAILS_VERSION', ['>= 4.1', '< 5.3'])
-  spec.add_dependency 'activesupport', ENV.fetch('RAILS_VERSION', ['>= 4.1', '< 5.3'])
+  spec.add_dependency 'activerecord', '>= 4.2', '< 6.1'
+  spec.add_dependency 'activesupport', '>= 4.2', '< 6.1'
 
+  spec.add_development_dependency 'appraisal'
   spec.add_development_dependency 'coveralls'
   spec.add_development_dependency 'database_cleaner', '>= 1.2'
-  spec.add_development_dependency 'rake', '< 11.0'
-  spec.add_development_dependency 'rspec', '~> 2'
+  spec.add_development_dependency 'rake', '>= 12.0'
+  spec.add_development_dependency 'rspec', '~> 3.8.0'
```

Comment:
Consider relaxing or dropping the version spec here - `~> 3.8` seems fine; I'd even gamble on letting rspec float, since if there's a 4.0 that breaks this test suite it would likely be easy to fix.

Reply:
I'd rather keep this pinned to the 3.8.x release stream so any deprecations in later 3.x.x builds don't add to the news of builds for this gem. I also want to make sure our builds continue to run even when rspec 4.0 is released. It sucks when you're making a change to a gem and you have to do a bunch of upgrades unrelated to your change just to get the build passing again.

Reply:
Just adding an observation from json_models gemspec, ignoring the fact it never needed `rspec-rails`. Leaving the `rspec-rails` unpinned blew out the bundle time to 30 minutes because it tried an insane amount of permutations of gems. So probably worth trying to pin it to something.